oggsound/include/OgreOggISound.h

```diff
@@ -581,11 +581,11 @@ namespace OgreOggSound
 		 */
 		virtual void _notifyAttached(
 			Ogre::Node* node
-			#if OGRE_VERSION_MAJOR != 2
+			#if AV_OGRE_NEXT_VERSION < 0x20000
```

Comment:
```suggestion
			#if !AV_OGRE_NEXT_VERSION
```
please use this format here and elsewhere as otherwise on can read this as Ogre Next 1.x which is not a thing.

Reply:
fixed